index.html

```diff
@@ -136,6 +136,48 @@ <h2></h2>
         </li>
     </template>
 
+    <template id="results-modal-template">
+        
+        <div class='popUpScreen' id='resultsScreen'>
+
+            <div class='popUpTitleResultsArea' id='popUpTitleAreaResults'>
+                <h1 class='popUpTitle' id='popUpTitleResults'>GAME OVER</h1>
+            </div>
+
+            <div class='resultsDescriptionArea'>
+                <p id='resultsDescription'>During <span id='minutesTaken'>2</span> minutes you answered <span id='answersGiven'>20</span> questions.</p>
```

Comment:
We should show seconds, not minutes (bo tak mierzymy w quizPage'u)

Reply:
Can you also provide function that will fill the popup with data returned from `GAME_HANDLER.getResults(x)`?